rust-runtime/aws-smithy-json/src/deserialize/error.rs

```diff
@@ -20,73 +21,86 @@ pub enum ErrorReason {
     UnexpectedEos,
```

Comment:
EOS means "end of string" right?

Reply:
"End of stream", I think. Close enough either way.